Directory.Packages.props

```diff
@@ -29,7 +29,6 @@
     <PackageVersion Include="System.ServiceModel.Security" Version="4.5.3" />
     <PackageVersion Include="System.Xml.XDocument" Version="4.3.0" />
     <PackageVersion Include="System.Xml.XmlDocument" Version="4.3.0" />
-    <PackageVersion Include="Xamarin.Android.Support.CustomTabs" Version="28.0.0.3" />
     <PackageVersion Include="Xamarin.AndroidX.Browser" Version="1.4.0" PrivateAssets="All" />
```

Comment:
Should this be removed too? We add it in net6-android, but I didn't see anywhere where we use this namespace in code.